test/tools/fuzzer_common.h

```diff
@@ -0,0 +1,41 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <libdevcore/CommonIO.h>
+#include <libevmasm/Assembly.h>
+#include <libsolc/libsolc.h>
+#include <libevmasm/ConstantOptimiser.h>
+
+#include <libdevcore/JSON.h>
+
+#include <string>
+#include <sstream>
+#include <iostream>
+
+using namespace std;
+using namespace dev;
+using namespace dev::eth;
+
+string contains(string const& _haystack, vector<string> const& _needles);
```

Comment:
should not be in global namespace.

Reply:
Is that used in both .cpp files? Hm, perhaps CommonData.h could be a place to put it, but I'm not sure - the name is a little misleading and you have to look at the signature. Perhaps `containsAnyOf`? Then it could go to CommonData.h.

Reply:
Or `findAnyOf`?

Reply:
I'd vote for `findAnyOf` instead of `containsAnyOf`, since it returns a `string` and not a `bool` ;)

Reply:
done.

Reply:
> I'd vote for findAnyOf instead of containsAnyOf, since it returns a string and not a bool ;)

I'd +1 the vote for findAnyOf.

Reply:
How about this?

`string contains(string const& _haystack, vector<string> const& _needles)`

becomes

`std::string findAnyOf(std::string const& _haystack, std::vector<std::string> const& _needles)`

and moves from `fuzzer_common.cpp` to inside the `dev` namespace in `libdevcore/CommonData.h`?